cpp/src/filling/fill.cu

```diff
@@ -60,11 +61,38 @@ struct in_place_fill_range_dispatch {
   cudf::mutable_column_view& destination;
 
   template <typename T>
-  std::enable_if_t<cudf::is_fixed_width<T>(), void> operator()(cudf::size_type begin,
+  std::enable_if_t<cudf::is_fixed_width<T>() && not cudf::is_fixed_point<T>(), void> operator()(
+    cudf::size_type begin, cudf::size_type end, cudaStream_t stream = 0)
+  {
+    in_place_fill<T>(destination, begin, end, value, stream);
+  }
+
+  template <typename T>
+  std::enable_if_t<cudf::is_fixed_point<T>(), void> operator()(cudf::size_type begin,
                                                                cudf::size_type end,
                                                                cudaStream_t stream = 0)
   {
-    in_place_fill<T>(destination, begin, end, value, stream);
+    if (value.type().id() == cudf::type_id::DECIMAL32) {
+      auto s = cudf::make_numeric_scalar(cudf::data_type(cudf::type_id::INT32));
```

Comment:
Are you able to use `make_fixed_point_scalar` here?

Reply:
@codereport  I think we need int32-typed scalar to keep align with int32-typed column view. Acually, I am not familiar with cudf native.  If I am wrong, please correct me. Thank you!